README.md

```diff
@@ -8,6 +8,11 @@ Wowza server side modules.  Uses Gradle (with the Gradle wrapper) as the build t
 
   ./gradlew build
 
+Before running this command, you'll need to set the `WOWZA_HONEYBADGER_API_KEY` environment variable.
+You can get the API key from our [Honeybadger](https://www.honeybadger.io/) project (`DLSS-Wowza`).
+Note that you may need to ask to get added to this project as a user in order to obtain the key.
```

Comment:
"this project" --  "this honeybadger project" ?   (since i'm reading in the context of a project, "this project" is a bit ambiguous)